operators/src/processing/map_query.rs

```diff
@@ -44,7 +45,13 @@ where
             log::debug!("Query was rewritten to empty query. Returning empty / filled stream.");
             let s = futures::stream::empty();
 
-            Ok(SparseTilesFillAdapter::new_like_subquery(s, query, self.additional_data).boxed())
+            Ok(SparseTilesFillAdapter::new_like_subquery(
+                s,
+                query,
+                self.additional_data,
+                CacheExpiration::NoCache, // TODO: can we do better?
```

Comment:
Use the input?

Reply:
If we map query, we will get tiles back from which we can use the expiration?

Reply:
yes, but in this branch of the if-statement, the query couldn't be rewritten and we start from an empty stream. If I am not mistaken.

Reply:
Could you add this as a comment? This is still unclear to me.

Reply:
4a6cb5331c6f8b07933327ef42b8ae8c924bb066